src/App.css

```diff
@@ -240,3 +240,31 @@
   padding-left: 10px;
   padding-right: 10px;
 }
+
+.header {
+  background-image: url(https://talloiresnetwork.tufts.edu/wp-content/uploads/TCD-header1.jpg);
```

Comment:
try avoid using background image in the css, doesnt seem to show up on ios - have a look at the course page other cian did and copy that for the picture